### PR TITLE
added enable_ap.sh script

### PR DESCRIPTION
**Pull request by younesr1**

### CODE REVIEW

File: dev_notes.txt [hunk 1]
```diff
@@ -57,3 +57,11 @@ Ensure you've run all your source commands (ie. source devel/setup.bash). These
 export ROS_MASTER_URI=http://129.97.71.49:11311
 export ROS_IP=<your ip address in same subnet as movo1>
 
+# Enable Access Point on Movo2
+Execute scripts/enable_ap.sh
+You should see a new AP called movo
+Password is same as robohub password
```
Comment (FelixKHWong) [inline dev_notes.txt]: Do we run this on Movo and our own computer?

File: dev_notes.txt [hunk 2]
```diff
@@ -57,3 +57,11 @@ Ensure you've run all your source commands (ie. source devel/setup.bash). These
...
+The first time you connect to the AP, you must manually configure your static IP address
+Use the Settings GUI and set your static ip address in the subnet 10.66.171.XX
```
Comment (FelixKHWong) [inline dev_notes.txt]: Is this ubuntu?

**Review by FelixKHWong — DISMISSED** [review]

File: scripts/enable_ap.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -v
+set -e
+sudo nmcli c up Hotspot
```
Comment (le2tang) [inline scripts/enable_ap.sh]: Is it useful to use sudo to run scripts/enable_ap.sh and avoid calling sudo from inside the script?

File: dev_notes.txt [hunk 4]
```diff
@@ -57,3 +57,11 @@ Ensure you've run all your source commands (ie. source devel/setup.bash). These
...
+Note: aliasing won't work. Use direct IP addresses of movos
```
Comment (le2tang) [inline dev_notes.txt]: I see that the IP addresses aren't directly listed in the dev_notes.txt, should they be added for reference

**Comment by younesr1** [comment]
no need for ap script. done on bootup

**Pull request closed without merging**